tt_eager/tt_dnn/op_library/backward/backward_ops.cpp

```diff
@@ -501,6 +501,21 @@ std::vector<Tensor> relu_bw(const Tensor& grad, const Tensor& input, const Memor
     return operation::decorate_as_composite(__func__, _relu_bw)(grad, input, output_mem_config);
 }
 
+std::vector<Tensor> _hypot_bw(const Tensor& grad, const Tensor& input, const Tensor& other, const MemoryConfig& output_mem_config) {
+    std::vector<Tensor> grad_tensor;
+    Tensor hyp_result = hypot(input, other, output_mem_config);
+    Tensor result_recip = recip(hyp_result, output_mem_config);
+    Tensor grad_a = mul(grad, mul(input, result_recip, std::nullopt, output_mem_config), std::nullopt, output_mem_config);
```

Comment:
You should scope/deallocate hyp_result after result_recip calculation

Reply:
I have made changes to address your comment. Kindly review